src/odbc_connection.cpp

```diff
@@ -2737,6 +2737,8 @@ bind_buffers
     // bind depending on the column
     switch(column->DataType) {
 
+      column->is_long_data = false;
+
```

Comment:
I'd move this out of the switch statement. Although it might work, I'm not sure how standard that is.

Reply:
I added a comment, and that made me realize I should probably swap the if/else statements as well. I think the OR statement makes more sense than that previous AND statement